freqtrade/logger.py

```diff
@@ -0,0 +1,40 @@
+# pragma pylint: disable=too-few-public-methods
+
+"""
+This module contains the class for logger and logging messages
+"""
+
+import logging
+
+
+class Logger(object):
```

Comment:
We could also implement a `logging.Handler` for color support, it would look like this:

```
import logging

# declare handler
class NullHandler(logging.Handler):
    def emit(self, record):
        pass

```

```
import logging

# Use handler
h = NullHandler()
logging.getLogger("foo").addHandler(h)
logging.info("...")
```

Right now the Logger class feels more like a logging wrapper which duplicates data. I also think we should define the logging configuration at one place, but I think a simple function to call `basicConfig` should be enough.

Reply:
You read my mind. In the long term, my idea was to extend logging and switch to a colored logging. This class is mostly a placeholder at this stage. If too overkill, we can drop it.

Reply:
I think we can leave it this way and see how we do in the long term